privaterelay/templates/includes/banners.html

```diff
@@ -79,28 +79,32 @@
             </div>
             {% endif %}
           {% elif not user_profile.subdomain %}
-            <div class="mpp-choose-subdomain" id="mpp-choose-subdomain">
-              <img src="/static/images/choose-subdomain.png" alt="">
-              <div class="border"></div>
-              <div class="mpp-choose-subdomain-form">
-                <h2>{% ftlmsg 'banner-choose-subdomain-headline-aliases' %}</h2>
-                <p>{% ftlmsg 'banner-choose-subdomain-copy' %}</p>
+            <div class="c-notification-banner subdomain-banner" id="mpp-choose-subdomain">
+              <div class="subdomain-banner-copy">
+                <div class="c-notification-banner-label">Action step</div>
```

Comment:
It's in the design, so good to go, but... As a user this strings gives me a bit of anxiety, since I'm not sure what conclusions I need to draw from it and if there are other steps I need to take action on :P

Reply:
Same.